source/agora/utils/Test.d

```diff
@@ -235,12 +236,10 @@ public Transaction[] makeChainedTransactions (KeyPair key_pair,
 ///
 unittest
 {
-    import agora.consensus.data.Block;
-
     auto gen_key = WK.Keys.Genesis;
 
     /// should spend genesis block's outputs
-    auto txes = makeChainedTransactions(gen_key, null, 1);
+    auto txes = genesisSpendable().map!(txb => txb.sign()).array();
```

Comment:
I don't think the changes in these tests make sense. These tests are for the `makeChainedTransactions` function, not for `TxBuilder`.

When a future commit removes `makeChainedTransactions` then these unittests will also be removed. But if `makeChainedTransaction` is not removed yet in this PR then its tests should not be touched.